pkg/syncer/cnsoperator/controller/cnsregistervolume/cnsregistervolume_controller.go

```diff
@@ -639,3 +756,63 @@ func updateCnsRegisterVolume(ctx context.Context, client client.Client,
 	}
 	return err
 }
+
+func getPatchData(oldObj, newObj interface{}) ([]byte, error) {
```

Comment:
Can we put these functions in a common file called patch and import them? I guess we already have these in 2 other files.

Reply:
Done